tests/integrations/celery/integration_tests/test_celery_beat_cron_monitoring.py

```diff
@@ -52,6 +53,7 @@ def inner(propagate_traces=True, monitor_beat_tasks=False, **kwargs):
     return inner
 
 
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
```

Comment:
Those integration tests (that are more for documentation) are hanging in Python 3.6 when using the Python Docker container. I just deactivated them, because we will remove 3.6 support soon anyhow.